.github/workflows/mdl.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   mdl:
     runs-on: ubuntu-latest
     env:
-      BUNDLE_ONLY: mdl
+      BUNDLE_ONLY: default mdl
```

Comment:
Let me confirm what `BUNDLE_ONLY: default` means. If it installs every gem file in the Gemfile I do not think we need to have this line. Because it installs every gems anyway.

Reply:
@yahonda Thanks for your review! It seems we can use `BUNDLE_INCLUDE` instead, see [f8d35ee](https://github.com/rails/rails/pull/47855/commits/f8d35ee5d31a728c4104c76a05a7fa561e748e28).

Reply:
Ahh, actually include is redundant so we can just remove the environment variable from the action.